misc.sk

```diff
@@ -174,7 +212,15 @@ old music discs:
 	[music] (disc|record) ward = minecraft:record_ward
 	any [music] (disc|record) = music disc 11, music disc 13, music disc blocks, music disc cat, music disc chirp, music disc far, music disc mall, music disc mellohi, music disc stal, music disc strad, music disc wait, music disc ward
 
-new dyes:
+village and pillage update dyes:
```

Comment:
For this to work (the 1.14 one to properly override) it needs to be declared below the other dye sections